libcloud/compute/drivers/openstack.py

```diff
@@ -2129,7 +2131,8 @@ def _to_node(self, api_node):
                 vm_state=api_node.get("OS-EXT-STS:vm_state", None),
                 power_state=api_node.get("OS-EXT-STS:power_state", None),
                 progress=api_node.get("progress", None),
-                fault=api_node.get('fault')
+                fault=api_node.get('fault'),
```

Comment:
I can remove it, but trailing commas is generally considered good practice on multiline structures so it's easier to reorder and add elements without forgetting the comma.  Python just ignores it.  The only reason JSON doesn't support it is Internet Explorer at the time didn't support it, despite the Javascript spec specifically allowing it.

Anyway, I didn't mean to add it here, I just moved some stuff around and forgot to put it back to how it was.